src/devtools/DevToolsViewProvider.ts

```diff
@@ -0,0 +1,129 @@
+import * as vscode from "vscode";
+import { devtoolsEvents } from "./server";
+export class DevToolsViewProvider implements vscode.WebviewViewProvider {
+  public static readonly viewType = "vscode-apollo-client-devtools";
+
+  constructor(private readonly _extensionUri: vscode.Uri) {}
+  async resolveWebviewView(
```

Comment:
```suggestion
import { devtoolsEvents } from "./server";

export class DevToolsViewProvider implements vscode.WebviewViewProvider {
  public static readonly viewType = "vscode-apollo-client-devtools";

  constructor(private readonly _extensionUri: vscode.Uri) {}
  
  async resolveWebviewView(
```

Just for readability sake, do you mind if we add a touch of space here?